tags/tags.toml

```
@@ -1211,10 +1211,19 @@ discord.js does not support features until they are officially documented.
 While there might be a way for these features to work, it can change at any time without any notice 
 """
 
-[partial-group-dm]
-keywords = ["group-dm", "send-does-not-exist"]
+[ephemeral] 
+keywords = ["ephemeral-deprecated"]
 content = """ 
-Bots cannot send messages in partial group dm channels.
-Use `if ('send' in channel)` or compare `channel.type` to mitigate this.
-A fix will be implemented in 14.16.2, where you can use `channel.isSendable()`
+The `ephemeral` option will be removed
```

Needs more context, probably something like this

```suggestion
The `ephemeral` option when sending a message will be removed in v15
```